src/client/results/GoogleMap.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+
+export default React.createClass({
```

Comment:
I can't see a reason why you're using old syntax ? is there one ?

ie react.creatClass instead of class extends Component, using the function keyword instead of just functionName(){} ?